consul/resource_consul_service_test.go

```diff
@@ -26,6 +26,7 @@ func TestAccConsulService_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("consul_service.example", "port", "80"),
 					resource.TestCheckResourceAttr("consul_service.example", "tags.#", "1"),
 					resource.TestCheckResourceAttr("consul_service.example", "tags.0", "tag0"),
+					resource.TestCheckResourceAttr("consul_service.example", "meta.test", "test"),
```

Comment:
I added this believing it would be checked, but when I set the value to `fail`, the test still passes. What exactly is this whole portion doing?

Reply:
If I had read the README.md, I would have understood that the test does do what I thought it did but only if there is a consul server available for the tests to run against.